### DISCUSSION TITLE
How to use Colored Image - With Mask

### DISCUSSION THREAD

**Discussion by creeduk** [melMass/comfy_mtb#139] [Q&A] [answered]
When ever I add a mask even one generated by an mtb node I get - 'builtin_function_or_method' object is not subscriptable. Is this a bug or am I doing something wrong?

**Comment by melMass** [comment]
Can you share the workflow that generates that? I'm curious of what combo does that.

Here is the workflow (embedded in the image):
![colored_image_example](https://github.com/melMass/comfy_mtb/assets/7041726/8d5305e3-1b67-4763-b70f-ccec32ce5bbc)

**Reply by melMass** [reply]
You meant mask! Yes I can reproduce too let me look into it!

**Comment by melMass** [comment, marked as answer]
So I fixed it in 30c4311b69f6481a34f968cb67a9b5ce5d2e9fda and also added a way to have mismatch in the provided size and images (it will crop or expand instead of erroring out), let me know if it works for you:

![colored_image_mask_example](https://github.com/melMass/comfy_mtb/assets/7041726/1c24c716-1be6-416a-a1c2-17792ce3e9f2)

**Reply by creeduk** [reply]
That works, my example was a logo and it already had an alpha channel so the image node made it al 1 solid color, the mask had that error. Now the mask works. 
I was able now also to load Mask as a separate node and invert etc. play around with the masking (convert image to mask). So working now thank you.
